src/Accounts/Accounts/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 ## Upcoming Release
 * Fixed typo in comments
+* Fixed an issue that may cause Az to skip logs in Azure Automation [#11492]
```

Comment:
Not only in Automation. Please double check comments
"I have a similar scenario - but we don't use Azure Automation. This is perfectly reproducable in plain PowerShell whenever you use powershell jobs."